beacon-chain/cache/committee.go

```diff
@@ -67,6 +68,9 @@ func NewCommitteeCache() *CommitteeCache {
 // ShuffledIndices fetches the shuffled indices by epoch and shard. Every list of indices
 // represent one committee. Returns true if the list exists with epoch and shard. Otherwise returns false, nil.
 func (c *CommitteeCache) ShuffledIndices(epoch uint64, shard uint64) ([]uint64, error) {
+	if !featureconfig.Get().EnableShuffledIndexCache {
```

Comment:
the committee cache is already gated by the new-cache flag if I am not wrong. 
@terencechain can you confirm ?

Reply:
that's correct, we don't need this flag

Reply:
Then that flag needs to be moved here. Not putting the flag inside of the cache allows for misuse without proper flag control.